module/VuFind/src/VuFind/Sitemap/Sitemap.php

```diff
@@ -38,13 +38,22 @@
  */
 class Sitemap extends AbstractFile
 {
+    const XHTML_NAMESPACE = 'xmlns:xhtml="http://www.w3.org/1999/xhtml"';
```

Comment:
We should set this public or protected to keep PHPStan happy when we raise its level in the future.

Reply:
Done.